### PR TITLE
generate-docs

### PR DESCRIPTION
**Pull request by iljarotar**

### CODE REVIEW

File: proto/validate/validate.proto [hunk 1]
```diff
@@ -0,0 +1,862 @@
+syntax = "proto2";
```
Comment (majst01) [inline proto/validate/validate.proto:1]: isnt it possible to define this dependency in buf.gen.yaml ?

Comment (iljarotar) [reply proto/validate/validate.proto:1]: I think it already is, but protoc gave an error.

Comment (majst01) [reply proto/validate/validate.proto:1]: OK, this is bad, we should also check if docs.json can be built with `buf`

File: proto/Makefile [hunk 2]
```diff
@@ -7,3 +7,7 @@ protolint:
 .PHONY: protoc
 protoc: protolint
 	buf generate -v
+
+PHONY: generate-docs
+generate-docs:
+	protoc --doc_out=../docs --doc_opt=json,docs.json --experimental_allow_proto3_optional $(wildcard ./api/v1/*.proto)
```
Comment (majst01) [inline proto/Makefile:13]: should be done with `buf`

**Review by majst01 — CHANGES_REQUESTED** [review]

File: proto/Makefile (hunk 2 above)
Comment (iljarotar) [reply proto/Makefile:13]: Is there a way to generate a json description with buf? I want to integrate the docs into the website from a json-file and I haven't found a way to generate it with buf, that's why I used protoc.

Comment (majst01) [reply proto/Makefile:13]: evtl mit einem davon: https://buf.build/search?query=protoc-gen-doc

**Comment by majst01** [comment]
closed as discussed with @iljarotar

**Pull request closed without merging**